v4/source/groups.yaml

```diff
@@ -1,3 +1,563 @@
+        '403':
+          $ref: '#/responses/Forbidden'
+        '501':
+          $ref: '#/responses/NotImplemented'
+        '500':
+          $ref: '#/responses/InternalServerError'
+  
+  '/groups/{group_id}/channels/{channel_id}':
+    get:
+      tags:
+        - groups
+      summary: Get GroupSyncable from channel ID
+      description: |
+        Get the GroupSyncable object with group_id and channel_id from params
+        ##### Permissions
+        Must have `manage_system` permission.
+
+        __Minimum server version__: 5.11
+      parameters:
+        - name: group_id
+          in: path
+          description: Group GUID
+          required: true
+          type: string
+        - name: channel_id
+          in: path
+          description: Channel GUID
+          required: true
+          type: string
+
+      responses:
+        '200':
+          description: GroupSyncable object retrieval successful
+          schema:
+            $ref: '#/definitions/GroupSyncableChannel'
+        '400':
+          $ref: '#/responses/BadRequest'
+        '401':
+          $ref: '#/responses/Unauthorized'
+        '403':
+          $ref: '#/responses/Forbidden'
+        '501':
+          $ref: '#/responses/NotImplemented'
+        '500':
+          $ref: '#/responses/InternalServerError'
+
+  '/groups/{group_id}/teams':
+    get:
+      tags:
+        - groups
+      summary: Get group teams
+      description: |
+        Retrieve the list of teams associated to the group
+        ##### Permissions
+        Must have `manage_system` permission.
+
+        __Minimum server version__: 5.11
+      parameters:
+        - name: group_id
+          in: path
+          description: Group GUID
+          required: true
+          type: string
+
+      responses:
+        '200':
+          description: Teams list retrieval successful
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/GroupSyncableTeams'
+        '400':
+          $ref: '#/responses/BadRequest'
+        '401':
+          $ref: '#/responses/Unauthorized'
+        '403':
+          $ref: '#/responses/Forbidden'
+        '501':
+          $ref: '#/responses/NotImplemented'
+        '500':
+          $ref: '#/responses/InternalServerError'
+  
+  '/groups/{group_id}/channels':
+    get:
+      tags:
+        - groups
+      summary: Get group channels
+      description: |
+        Retrieve the list of channels associated to the group
+        ##### Permissions
+        Must have `manage_system` permission.
+
+        __Minimum server version__: 5.11
+      parameters:
+        - name: group_id
+          in: path
+          description: Group GUID
+          required: true
+          type: string
+
+      responses:
+        '200':
+          description: Channel list retrieval successful
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/GroupSyncableChannels'
```

Comment:
Same thing here:
```suggestion
              $ref: '#/definitions/GroupSyncableChannel'
```

Reply:
Same thing as above.
`/groups/:team_id/channels/:channel_id`  
![image](https://user-images.githubusercontent.com/17804942/67727886-819c0180-f9c1-11e9-8de4-ce9808920aa0.png)

`/groups/:team_id/channels`  
![image](https://user-images.githubusercontent.com/17804942/67727896-924c7780-f9c1-11e9-92dc-e72415344740.png)  

Discrepancy in information received from each call hence the different definitions.